spec/services/duty_options/quota/non_preferential_end_use_spec.rb

```diff
@@ -14,7 +14,7 @@
         value: 96,
         order_number: '058048',
         measure_sid: measure.id,
-        source: nil,
+        source: 'uk',
```

Comment:
If you're curious, this changed because I was properly serializing the measures in the factory (`measure#source`) gets called so we can know where the calculation comes from for a given duty option